tests/test_synctest_session_enum.rs

```diff
@@ -0,0 +1,25 @@
+mod stubs_enum;
+
+use ggrs::{GGRSError, SessionBuilder};
+
+#[test]
+fn test_enum_advance_frames_with_delayed_input() -> Result<(), GGRSError> {
```

Comment:
as said in comment, I would have like to provide more tests, but it's creating a lot of duplicated code, so I figured we could merge minimal tests, and then discuss a better test strategy:
- I tried a generic `GameStub`, but all the data structures are quite tied, and the hardest to make generic is the `Config` implementation.
- macros could be another idea, but they can be controversial.